CI/Dockerfile

```
@@ -169,47 +157,76 @@ ENV moab_install_dir=${install_dir}/moab
 
 RUN mkdir -p ${moab_build_dir}/build && \
     cd ${moab_build_dir} && \
-    git clone -b ${MOAB_BRANCH} --depth 1 https://bitbucket.org/fathomteam/moab && \
+    git clone -b ${MOAB_VERSION} --depth 1 https://bitbucket.org/fathomteam/moab && \
     cd build && \
-    cmake ../moab -DCMAKE_INSTALL_RPATH=${hdf5_install_dir}/lib:${moab_install_dir}/lib \
-                  -DCMAKE_BUILD_TYPE=Release \
-                  -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
-                  -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
-                  -DCMAKE_CXX_COMPILER=${CXX} \
-                  -DCMAKE_C_COMPILER=${CC} \
-                  -DENABLE_BLASLAPACK=OFF \
-                  -DBUILD_SHARED_LIBS=ON \
-                  -DENABLE_FORTRAN=OFF \
-                  -DENABLE_PYMOAB=ON && \
-    make -j${ci_jobs} && \
+    cmake ../moab \
+        -DCMAKE_INSTALL_RPATH=${hdf5_install_dir}/lib:${moab_install_dir}/lib \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
+        -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
+        -DCMAKE_CXX_COMPILER=${CXX} \
+        -DCMAKE_C_COMPILER=${CC} \
+        -DENABLE_BLASLAPACK=OFF \
+        -DBUILD_SHARED_LIBS=ON \
+        -DENABLE_FORTRAN=OFF \
+        -DENABLE_PYMOAB=ON && \
+    make -j${CI_JOBS} && \
     make install && \
     cd && \
     rm -rf ${moab_build_dir}
 
+ENV embree_install_dir=${install_dir}/embree
+ENV embree_build_dir=${build_dir}/embree
+
+# Clone and install Embree
+# Required for Double-Down
+RUN if [ "${DOUBLE_DOWN}" = "ON" ]; then \
```

With previous comments this becomes:
```suggestion
RUN if [ "${DOUBLE_DOWN_VERSION}" != "OFF" ]; then \
```